en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/M/Melestruas_Marvelous_Marquee.md

```diff
@@ -14,7 +14,8 @@
 [_metadata_:components_material_description]:- "a small crystal bead"
 [_metadata_:concentration]:- "false"
 [_metadata_:duration]:- "8 hours"
-[_metadata_:spell_origin]:- "wotc_srd_5.1"
+[_metadata_:compared_to_wotc_srd]:- "mechanics_same_wording_same"
+[_metadata_:compared_to_a5e_srd]:- "mechanics_different_wording_different"
```

Comment:
This actually surprised me; the A5e version doesn't block creatures from moving into the area.